pkg/agent/core/ngt/handler/grpc/handler.go

```diff
@@ -1047,7 +1047,7 @@ func (s *server) MultiUpsert(ctx context.Context, reqs *payload.Upsert_MultiRequ
 }
 
 func (s *server) Remove(ctx context.Context, req *payload.Remove_Request) (res *payload.Object_Location, err error) {
-	ctx, span := trace.StartSpan(ctx, apiName+".Remove")
+	_, span := trace.StartSpan(ctx, apiName+".Remove")
```

Comment:
StartSpan often generates a new context by context.WithValue with embedded traceID and other values in context. I think it is better to ignore this part even CodeQL warns it.